assembl/static2/js/app/components/debate/multiColumns/columnsView.jsx

```diff
@@ -53,13 +64,30 @@ class ColumnsView extends React.Component {
       return countSynthesis >= 1;
     });
 
+    const { slug, phase, themeId } = routerParams;
+    const openShareSynthesisModal = (id) => {
```

Comment:
There was a method to `openSharePostModal`. Now you're adding specific method for this. Looking quickly, there isn't much of a difference between the two. This is often a good practice to abstract this operation to a method that is independent of the type, perhaps put it on the highest level component, or as a function in the `utils` or pass it as a prop.

Reply:
okay, I just rename both openShareModal ?

Reply:
I'm referring to writing one method and using it across all components. If that is what you're referring to, then yes an `openShareModal`.

Reply:
Done.